virt/virt-web-console-overview.adoc

```diff
@@ -347,43 +347,43 @@ By default, Red Hat templates are stored in the `openshift` project. You can edi
 |====
 =====
 
-// .*InstanceTypes* tab
-// [%collapsible]
-
-// =====
-// [cols="1,3a", options="header"]
-// |====
-// |Element
-// |Description
+[id="virtualization-instancetype_{context}"]
+=== InstanceTypes tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element
+|Description
 
-// |*Volumes project* list
-// |Select the volume to boot your VM from.
+|*Volumes project* list
+|Select the volume to boot your VM from.
```

Comment:
This definition belongs with a different UI element, "Volume table". 

The **Volume** table displays a list of ....